packages/typescriptlang-org/src/components/layout/main.scss

```diff
@@ -72,6 +72,13 @@ html {
   }
 }
 
+// Style the permalink icon in dark colour-scheme
+.raised svg path {
```

Comment:
I think the s probably too loose (it would apply to almost every SVG in the site!) - if you can narrow it down to something like `.raised .markdown h1 a svg path, .raised .markdown h2 a svg path, .raised .markdown h3 a svg path` ?

Reply:
Thanks, done.